utils/curl.xml

```diff
@@ -283,13 +283,13 @@
       <manifest-digest sha256new="WPBA7YQ2UXZ4L3X4S6WGEYHNOGLDB6AIR2UJYWG2Q2V2V2JBGXQQ"/>
       <archive extract="curl-7.63.0-win32-mingw" href="https://bintray.com/vszakats/generic/download_file?file_path=curl-7.63.0-win32-mingw.tar.xz" size="2049216" type="application/x-xz-compressed-tar"/>
     </implementation>
-    <implementation arch="Windows-x86_64" id="sha1new=1ed19054d4ac2157074091abdc78825379dd818b" released="2019-02-06" stability="stable" version="7.64.0">
```

Comment:
Should the automatic-updater be changing existing implementations like this?

Reply:
I made this change manually after I noticed that the upstream site had changed the content of the archive. Unfortunately they did this without changing the URL or version number. I'm assuming they fixed some packaging issue without rebuilding the binaries.